### PR TITLE
Define Utility Functions

### PR DESCRIPTION
**Pull request by aamederen**
This change declares two new functions:

`master_update_table_statistics` updates the statistics of shards belong
to the given table as well as its colocated tables.

`get_colocated_shard_array` returns the ids of colocated shards of a
given shard.

### CODE REVIEW

File: src/backend/distributed/citus--7.0-10--7.0-11.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		logicalrelid = ANY (colocated_tables);
+END;
+$$ LANGUAGE 'plpgsql';
+COMMENT ON FUNCTION master_update_table_statistics(regclass)
+	IS 'updates shard statistics of the given table and its colocated tables';
+	
+CREATE OR REPLACE FUNCTION get_colocated_shard_array(bigint)
+	RETURNS BIGINT[]
+	LANGUAGE C STRICT
+	AS 'citus', $$get_colocated_shard_array$$;
```
Comment (marcocitus) [inline src/backend/distributed/citus--7.0-10--7.0-11.sql]: Could we move the definition of this function out of the test folder?

File: src/backend/distributed/test/metadata_sync.c [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include "fmgr.h"
 
 #include "catalog/pg_type.h"
+#include "distributed/listutils.h"
 #include "distributed/metadata_sync.h"
 #include "distributed/test_helper_functions.h" /* IWYU pragma: keep */
```
Comment (marcocitus) [inline src/backend/distributed/test/metadata_sync.c]: Well, this was messy.

Maybe remove some of the `#include "distributed/test_helper_functions.h"` lines.

Comment (aamederen) [inline src/backend/distributed/test/metadata_sync.c]: Removed all #includes of `test_helper_functions.h` and didn't get any complains. Weird.

**Comment by aamederen** [comment]
@marcocitus Any other comments?

**Pull request merged**